changelog/28121.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:improvement
-auth/okta: update to okta sdk v5
+auth/okta: update to okta sdk v5 from v2. Transitively updates go-jose dependency to >=3.0.3. See https://github.com/okta/okta-sdk-golang/blob/master/MIGRATING.md for details on changes.
```

Comment:
```suggestion
auth/okta: update to okta sdk v5 from v2. Transitively updates go-jose dependency to >=3.0.3 to remove security vulnerability. See https://github.com/okta/okta-sdk-golang/blob/master/MIGRATING.md for details on changes.
```

Reply:
I think it has some value for mentioning the purpose of the version bump before diving into the original PR.

Reply:
Added a reference to GO-2024-2631 (the vuln in `go-jose`)